vllm/engine/llm_engine.py

```diff
@@ -116,7 +117,8 @@ def __init__(
     def _init_workers(self, distributed_init_method: str):
         # Lazy import the Worker to avoid importing torch.cuda/xformers
         # before CUDA_VISIBLE_DEVICES is set in the Worker
-        from vllm.worker.worker import Worker  # pylint: disable=import-outside-toplevel
+        from vllm.worker.worker import \
+            Worker  # pylint: disable=import-outside-toplevel
```

Comment:
Please avoid using `\` for line continuation per [google python style guide](https://google.github.io/styleguide/pyguide.html#32-line-length)
```suggestion
        from vllm.worker.worker import Worker  # pylint: disable=import-outside-toplevel
```